app/models/journeys/additional_payments_for_teaching.rb

```diff
@@ -30,6 +30,14 @@ module AdditionalPaymentsForTeaching
       }
     }.freeze
 
+    def policies
+      if FeatureFlag.enabled?(:tri_only_journey)
+        [Policies::EarlyCareerPayments]
+      else
+        POLICIES
+      end
+    end
```

Comment:
We have to do this as the code doesn't support a Policy (`TargetedRetentionIncentivePayments`) being present in multiple journeys, there's various places where we look up the journey for a given policy. When this feature flag is enabled the additional payments journey only allows claiming for EY. This isn't an issue as we're not accepting AP claims, and in the PRs to removed additional payments we remove this journey and the usage of the feature flag.
We've introduced the feature flag to avoid doing too much in this already large PR!

Reply:
Yeah, TMA (too many acronyms)